submissions/AnzhelikaKh/popup_task/index.html

```diff
@@ -0,0 +1,121 @@
+                            <img src="./img/popup-button.png" alt="popup-button">
+                        </label>
+                        <ul class="popup__list">
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img src="./img/apps.png" alt="Apps">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img src="./img/apps.png" alt="Apps">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img src="./img/apps.png" alt="Apps">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img src="./img/apps.png" alt="Apps">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img src="./img/apps.png" alt="Apps">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="item__link">
+                                    <div class="item__view">
+                                        <img src="./img/apps.png" alt="Apps">
+                                        <p>Apps</p>
+                                    </div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <input type="checkbox" class="checkbox__more" id="checkbox__more">
+                                <label for="checkbox__more" class="more__label">
```

Comment:
Logically this button must be hidden when the second part of the popup is open.

Reply:
Still not fixed :cry:

Reply:
Is the option when the "More" button is hidden and changed to the "Less" button not suitable? 😢

Reply:
Holding the 'More/less' button in the middle of the list doesn't look good. Usually, it keeps after list.

Reply:
Okey, uno momento! ^^